CHANGES.rst

```diff
@@ -1268,7 +1268,9 @@ astropy.modeling
 astropy.nddata
 ^^^^^^^^^^^^^^
 
-- Fixed the bug in CCData.read when the HDU is not specified and the first one is empty so the function searches for the first HDU with data which may not have an image extension. [#7739]
+- Fixed the bug in CCData.read when the HDU is not specified and the first one
```

Comment:
This can be compacted a bit

Reply:
The original log entry was approved by @crawfordsm . Do you have a recommendation of alternate text?

Reply:
OK, then let's keep this.